ppa/archive/models.py

```diff
@@ -283,13 +361,16 @@ def populate_from_bibdata(self, bibdata):
         '''
         # store hathi record id
         self.record_id = bibdata.record_id
+        print(bool(bibdata.marcxml))
```

Comment:
looks like a leftover from debugging or testing something

Reply:
Thanks for the spot.